samples/Chat/src/app/ChatScreen.tsx

```diff
@@ -21,6 +21,11 @@ import { fetchEmojiForUser } from './utils/emojiCache';
 import { getBackgroundColor } from './utils/utils';
 import { useSwitchableFluentTheme } from './theming/SwitchableFluentThemeProvider';
 
+/* @conditional-compile-remove(attachment-upload) */
```

Comment:
Don need to add extra space here - one day we will remove all CC comments, and this extra line wouldn't be remove 

So the trick is try to ignore the existence of CC and imagine how it looks when it gets removed

Reply:
thanks removed